Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -156,6 +156,8 @@ extension StripeCardReaderService: CardReaderService {
             }
 
             Terminal.shared.collectPaymentMethod(activeIntent, delegate: self) { (intent, error) in
+                self?.sendReaderEvent(.cardRemoved)
```

Comment:
Please add a comment explaining why we are sending this event and if it applies to swiped or tapped cards (or only inserted ones)